x/mongo/driver/compression.go

```diff
@@ -111,6 +111,12 @@ func DecompressPayload(in []byte, opts CompressionOpts) (uncompressed []byte, er
 	case wiremessage.CompressorNoOp:
 		return in, nil
 	case wiremessage.CompressorSnappy:
+		len, err := snappy.DecodedLen(in)
+		if err != nil {
+			return nil, fmt.Errorf("error decoding compressed length %v", err)
```

Comment:
Drop "error" prefix, use "%w".

```suggestion
			return nil, fmt.Errorf("decoding compressed length %w", err)
```